docs/governance/governance.md

```diff
@@ -48,7 +48,35 @@ The “call for objections” or the vote must be conducted during a live TSC me
 
 ### Maintainers: <a href="#maintainers" id="maintainers"></a>
 
-Contributors who have the ability to commit code and contributions to a repo's main branch on the Baseline Protocol. A Contributor may become a Maintainer by a majority approval of the existing Maintainers. The initial number of maintainers required to merge a pull request to master in the github repo shall be three, but may be amended to no fewer than two by a simple majority of the maintainers.
+Baseline Maintainers have advanced code repository permissions and responsibilities. 
+
+#### Becoming a Baseline Maintainer
+
+First, one must attend at least two consecutive Baseline Core Dev meetings and contribute to the project through PRs, workgroups, or other contributions.
```

Comment:
even though this can not be precise anyways, it still seems a bit subjective
is it left to discretion of current maintainers to decide based on candidate contributions? i think since maintainer is usually technical person that will need to review the code as most important responsibility, probably only technical contributions should be taken in consideration?

Reply:
Good point. This will be added to future TSC & Core Dev discussions.